packages/strapi-admin/admin/src/translations/sv.json

```diff
@@ -0,0 +1,157 @@
+{
+  "Analytics": "Analytics",
+  "Content Manager": "Innehållshantering",
+  "Content Type Builder": "Innehållstyp Byggare",
+  "Email": "E-post",
+  "Files Upload": "Uppladdning av filer",
+  "HomePage.notification.newsLetter.success": "Prenumererade på nyhetsbrevet",
+  "New entry": "Nytt inlägg",
+  "Password": "Lösenord",
+  "Provider": "Leverantör",
+  "ResetPasswordToken": "Återställ lösenordstoken",
+  "Role": "Roll",
+  "Roles & Permissions": "Roller och behörigheter",
+  "Settings Manager": "Inställningshanterare",
+  "Username": "Användarnamn",
+  "Users": "Användare",
+  "Users & Permissions": "Användare och behörigheter,
+  "app.components.BlockLink.code": "Kodsexempel",
+  "app.components.BlockLink.code.content": "Lär dig genom att testa verkliga projekt utvecklade samhället.",
+  "app.components.BlockLink.documentation": "Läs dokumentationen",
+  "app.components.BlockLink.documentation.content": "Upptäck koncept, referensguider och handledning.",
+  "app.components.Button.cancel": "Avbryt",
+  "app.components.Button.save": "Spara",
+  "app.components.ComingSoonPage.comingSoon": "Kommer snart",
+  "app.components.ComingSoonPage.featuresNotAvailable": "Denna funktion är fortfarande under aktiv utveckling.",
+  "app.components.DownloadInfo.download": "Nedladdning pågår ...",
+  "app.components.DownloadInfo.text": "Det kan ta en minut. Tack för ditt tålamod.",
+  "app.components.EmptyAttributes.title": "Det finns inga fält än",
+  "app.components.HomePage.button.blog": "SE MER PÅ BLOGGEN",
+  "app.components.HomePage.button.quickStart": "STARTA HURTIGT START TUTORIAL",
+  "app.components.HomePage.community": "Hitta gemenskapen på webben",
+  "app.components.HomePage.community.content": "Diskutera med teammedlemmar, bidragsgivare och utvecklare på olika kanaler.",
+  "app.components.HomePage.create": "Skapa din första innehållstyp",
+  "app.components.HomePage.createBlock.content.first": "De",
+  "app.components.HomePage.createBlock.content.second": "plugin hjälper dig att definiera datastrukturen för dina modeller. Om du är ny här rekommenderar vi dig starkt att följa vår",
+  "app.components.HomePage.createBlock.content.tutorial": "handledning.",
+  "app.components.HomePage.cta": "BEKRÄFTA",
+  "app.components.HomePage.newsLetter": "Prenumerera på nyhetsbrevet för att kontakta Strapi",
+  "app.components.HomePage.support": "STÖD OSS",
+  "app.components.HomePage.support.content": "Genom att köpa en T-shirt tillåter det oss att fortsätta vårt arbete med projektet för att ge dig bästa möjliga upplevelse! ",
+  "app.components.HomePage.support.link": "FÅ DIN T-SHIRT NU",
+  "app.components.HomePage.welcome": "Välkommen ombord!",
+  "app.components.HomePage.welcome.again": "Välkommen ",
+  "app.components.HomePage.welcomeBlock.content": "Vi är glada över att ha dig som en del av samhället. Vi letar ständigt efter feedback så skicka oss DM på ",
```

Comment:
The use of "samhället" in place of community is really weird. "Nätforum" or plainly "communityn" would sound better.

Reply:
> The use of "samhället" in place of community is really weird. "Nätforum" or plainly "communityn" would sound better.

Changed.